openpype/settings/defaults/project_settings/houdini.json

```diff
@@ -21,6 +21,12 @@
         "file_rules": {
             "activate_host_rules": false,
             "rules": {}
+        },
+        "workfile":{
+            "enabled": false,
+            "default_display": "ACES",
+            "default_view": "sRGB",
+            "review_color_space": ""
```

Comment:
> > Review colorspace -> the colorspace name for the current default_display and default_view
>
> Can we add placeholder text in this fields when empty? Do text fields in settings support that? That might help clarify what type of data to enter.

@BigRoy 

I thought about adding a default value here but I didn't as I can't guarantee that users won't change any of the default display and view. 

I remember we discussed this before.
Do you prefer moving this field to `CreateReview` for more clarity ?

Reply:
I still think that'd make more sense if I compare it to how it's done for the majority of other code in the codebase - whether it's logical for a newcomer to AYON I'm not sure. I'lll leave this up to you.